webapp/src/app/shared/sdk/lb.config.ts

```diff
@@ -20,23 +20,23 @@
 * }
 **/
 export class LoopBackConfig {
-  private static path: string = '//0.0.0.0:3000';
+  private static path: string = '//localhost:3000';
```

Comment:
Let's not adjust the SDK - this will be un-done when the it's regenerated. 

The correct way to set the default is to make sure it's correct here: https://github.com/beeman/fireloop-starter/blob/develop/webapp/src/app/app.component.ts#L40